[Feature] #59 Security configuration, Jwt 세팅

sy-k-98 opened the pull request:
close #59 

- 토큰 발급
- security 필터 설정
- 로그인된 사용자 불러오기


기능이 잘 되는지는 회원가입, 로그인 다 구현하고 다시 확인해보겠습니다.

원래는 username으로 검색하는데 저희는 이메일로 로그인할거라 이메일로 로그인된 사용자를 불러오는 것으로 햇슴다

sy-k-98 committed:
feat : Security, Jwt 세팅
- 토큰 발급
- security 필터 설정
- 로그인된 사용자 불러오기

greedyBackEnd reviewed (commented):
로그인 email인 user_email이 username으로 가고,
별도로 email, realname으로 따로 칼럼으로 받게 수정해주시면 100번 감사드리곘습니다!!
== 현재 로그인 유저 정보 => 현재 유저의 username 정보를 받아오게입니다!

sy-k-98 commented:
내일 하겠습니다요..

sy-k-98 committed:
feat : Security, Jwt 세팅
- userEmail -> username으로 수정
- username -> realname으로 수정
- CustomUserDetailsService -> JpaUserDetailsManager로 수정

yonzn0213 commented:
이메일은 아예 없어진건가용?

sy-k-98 commented:
username이 이메일이 되는거 아닌가요??
username, email 두개 똑같은데 따로 있어야되는건가..?

yonzn0213 commented:
아 변수명만 바뀐건가용 따로 이메일 컬럼이 생기는 줄 알았습니다!!

greedyBackEnd reviewed (commented):
너무 좋아요! 감사합니다!

greedyBackEnd committed:
Merge branch 'develop' into #59-security-config
